dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -720,23 +723,50 @@ public ProcessMeta exportProcessMetaData(ProcessDefinition processDefinition) {
     }
 
     /**
-     * correct task param which has datasource or dependent
+     * correct task param which has datasource or dependent for export
```

Comment:
This method seems to inject some parameters into the TaskNode, I think the comment may be not clear enough.

And if other parameters will be injected in the future, it is better to split this method into two separate method.

Reply:
Thx, I'll fix comment.
This is used for import and export process. In the future, if there are inject other parameters, you can refactor again.